### ISSUE TITLE
noqa not working

### ISSUE THREAD

**Issue by GrbavaCigla** [PyCQA/pylint#4103]
I have django project and in the end i have these lines:
```python
try:
    from .local_settings import * # noqa
except ImportError:
    print('Local settings not found!')
```
`local_settings.py` can exist, but it doesn''t have to.
When I put `# noqa` at the end of the line I still get this warning:
```
pitv/pitv/settings.py:146:4: W0614: Unused import SECRET_KEY from wildcard import (unused-wildcard-import)
```
If you want more info, here is the django project: https://github.com/PiSmartTV/Website

**Comment by Pierre-Sassoulas** [comment]
`noqa` are not the way yo disable a message in pylint you should use this:
```python
try:
    from .local_settings import * # pylint: disable=unused-wildcard-import
except ImportError:
    print('Local settings not found!')
```
